facts/inventory.py

```diff
@@ -192,20 +192,11 @@ def dhcp4ranges(prefix, bitmask):
         return ["", "", "", "", ""]
     ipsplit = re.split(r"\.", prefix)
     if bitmask == 24:
-        # FIX: Hardcoding the hiInfra for larger pool to handle aplist.csv
-        # not accounting for the 2 buildings since hilton is solo
-        if prefix == "10.0.3.0":  # pylint: disable=R1705
-            return [
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".150",
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".250",
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".1",
-            ]
-        else:
-            return [
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".80",
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".254",
-                ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".1",
-            ]
+        return [
+            ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".80",
+            ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".254",
+            ipsplit[0] + "." + ipsplit[1] + "." + ipsplit[2] + ".1",
+        ]
```

Comment:
Rather than have a specific call out for 10.0.3.0/24 (and not a corresponding one for 10.128.3.0/24) I chose to just standardize this DHCP pool to what all other management vlans are. I think this simplifies things and supports what is being requested. The only exception to the standard pool is now non /24s (wifi guests) as has been the case for years.